test/built-ins/TypedArray/length-invalid.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.2.1.2
+description: TypedArray rejects non-integer arguments
+info: >
+  22.2.1.2 %TypedArray% ( length )
+
+  ...
+  4. Let numberLength be ToNumber(length).
+  5. Let elementLength be ToLength(numberLength).
+  6. ReturnIfAbrupt(elementLength).
+  7. If SameValueZero(numberLength, elementLength) is false, throw a RangeError
+  exception.
+  ...
+features: [TypedArray]
+includes: [testTypedArray.js]
+---*/
+
+assert.throws(RangeError, function() {
```

Comment:
It took me a while to think about this case and as I discussed with @jugglinmike: this is still a valid es2015 test flagged with a es6id.

I looked at the changes on the new specs but it's not published/released. So that makes a certainly confusing situation where this test is valid even if we have another file covering the expected behavior with a `es7id: pending` tag. 

From here, I believe the best case would be stick with this test and maybe add a note on info mentioning this is expected to change on es2016.